Detectors/ITSMFT/ITS/simulation/include/ITSSimulation/Detector.h

```diff
@@ -142,8 +142,9 @@ class Detector : public AliceO2::Base::Detector
                                     UInt_t &dettype) const;
 
     /// This method is an example of how to add your own point of type Point to the clones array
-    AliceO2::ITSMFT::Point *addHit(Int_t trackID, Int_t detID, TVector3 startPos, TVector3 pos, TVector3 mom, Double_t startTime,
-                  Double_t time, Double_t length, Double_t eLoss, Int_t shunt, Int_t status, Int_t statusStart);
+    AliceO2::ITSMFT::Point *addHit(int trackID, int detID, TVector3 startPos, TVector3 endPos, TVector3 startMom,
```

Comment:
unless needed, I think you should get rid of the momentum part here. Also, I would be in favour of not using TVector3 anymore. We have now Point3D + Vector3D .. or just pass the components individually.

TVector3 is not templated and will always use double values which means lots of casting etc.